docs/v3/source/includes/resources/manifests/_object.md.erb

```diff
@@ -139,7 +142,16 @@ will be updated for any existing route mapping.
 Name | Type | Description
 ---- | ---- | -----------
 **route** | _string_ | **(Required)** The route URI. Example: `host.domain.com`
-**protocol** | _string_ | Protocol to use for this route. Valid protocols are `http1`, `http2`, and `tcp`.
+**protocol** | _string_ | Protocol to use for this route. Valid protocols are `http1`, `http2`, and `tcp`
+**options** | _object_ | A key-value mapping of configuration [options](#space-manifest-per-route-options-level-configuration) to be applied to the specific application route
+
+####  <a name="space-manifest-per-route-options-level-configuration"></a> Per-Route Options-level configuration
+This configuration allows additional key-value options for the specific application route.
+
+| Name              | Type     | Description                                                                                         |
+|-------------------|----------|-----------------------------------------------------------------------------------------------------|
+| **loadbalancing** | _string_ | The load-balancer associated with this route. Valid values are ‘round-robin’ and ‘least-connection’ |
```

Comment:
The other tables do not have the surrounding `|` character, looks like some modified version of markdown tables, not sure if this will work.

Reply:
Adapted